packages/explorer/src/components/LatestBlock.tsx

```diff
@@ -7,18 +7,16 @@ export function LatestBlock() {
   const { data: block } = useBlockNumber({
     watch: true,
     chainId,
+    query: {
+      refetchInterval: 1000,
+    },
   });
 
   return (
     <div className="inline-block w-[50px]">
       {block ? (
         <div className="flex items-center justify-end text-xs font-extrabold text-green-600">
-          <span
-            className="mr-2 inline-block h-[8px] w-[8px] animate-pulse rounded-full"
-            style={{
-              background: "rgb(64, 182, 107)",
-            }}
-          ></span>
+          <span className="mr-2 inline-block h-[8px] w-[8px] animate-pulse rounded-full bg-success"></span>
```

Comment:
as a general rule and where possible, I try not to add spacing to individual elements to move siblings, but rather change this at the parent

since we already have `flex` on the parent, we can just add `gap-2` above to do the same kind of spacing here

Reply:
Yeah good point and generally try to follow the same logic, should be more mindful about it. Added gap property.